doc/source/getting_started/comparison/includes/filtering.rst

```diff
@@ -0,0 +1,25 @@
+:orphan:
+
+DataFrames can be filtered in multiple ways; the most intuitive of which is using
+:ref:`boolean indexing <indexing.boolean>`
+
+.. ipython:: python
+   :suppress:
+
+   # ensure tips is defined when scanning with flake8-rst
+   if 'tips' not in vars():
```

Comment:
i think you can just add this 
```
   url = (
       "https://raw.github.com/pandas-dev"
       "/pandas/master/pandas/tests/io/data/csv/tips.csv"
   )
   tips = pd.read_csv(url)
```

but maybe @jorisvandenbossche has a better soln

Reply:
But we would want to avoid to read this data multiple times in each file during the doc build, I think, so ideally we can let flake-rst ignore this in a different way

Reply:
> But we would want to avoid to read this data multiple times in each file during the doc build, I think, so ideally we can let flake-rst ignore this in a different way

sure, any idea how?